src/utils/handle_sound/handle_sound.h

```diff
@@ -1,8 +1,14 @@
 #pragma once
 #include <string>
+#include <vector>
 
 namespace utils
 {
+// The master volume of the application. It is an integer between 0 and 1000.
+extern int g_sound_volume;
+// A list of all the open audio files.
+extern std::vector<std::string> g_audio_list;
```

Comment:
As a note, opening (and playing) an audio file does increase our program's use of memory quite a lot. I noticed the following memory usage:
a) Without any open mp3 files: ~450KB
b) With one mp3 file open: ~3000KB
c) With two mp3 files open at the same time: ~3800KB
d) When all mp3 files have been closed, the memory stays at ~2000KB

Perhaps we should consider (as a future fix) limiting the number of concurrent open audio tracks, and have this vector be a simple array of fixed length. Again, your opinion here is much appreciated, @DIONYSUS-BASOUNAS .

Reply:
we have to consider other sound effects we may add in the future, like a sword swing, flesh slashing, and other little sound efects that would play during the battle